packages/bus/src/RetryStrategies/ExponentialBackoffRetryStrategy.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { ExponentialBackoffRetryStrategy } from './ExponentialBackoffRetryStrategy';
+import { instance, mock, reset, verify, when } from 'ts-mockito';
+
+describe('ExponentialBackoffRetryStrategy', () => {
```

Comment:
There are a couple of missed tests:

- should retry X times and throw an error
- should return a result execution immediately 
- should return a result execution after a few retries